src/apps/content-editor/src/app/views/ItemEdit/ItemEdit.js

```diff
@@ -335,6 +338,22 @@ export default function ItemEdit() {
                   />
                 )}
               />
+              <Route
+                exact
+                path="/content/:modelZUID/:itemZUID/publish-state"
```

Comment:
let's change this to "/publishings". This way it matches the API and avoids our development jargon "state"